hydra-node/src/Hydra/Chain/Direct/Tx.hs

```diff
@@ -196,10 +192,14 @@ initTx cardanoKeys HeadParameters{contestationPeriod, parties} txIn =
         & Api.addVkInputs [Api.fromLedgerTxIn txIn]
         & Api.addOutputs (headOutput : map mkInitialOutput cardanoKeys)
  where
-  headOutput = Api.TxOut headAddress headValue headDatum
-  headScript = Api.fromPlutusScript $ MockHead.validatorScript policyId
-  headAddress = Api.mkScriptAddress networkId headScript
-  headValue = Api.lovelaceToTxOutValue $ Api.Lovelace 2_000_000
+  headOutput =
+    Api.TxOut headAddress headValue headDatum
+  headScript =
+    Api.fromPlutusScript $ MockHead.validatorScript policyId
```

Comment:
Puzzle: Elsewhere we use `fromPlutusScript'`, can't remember the difference and whether or not it's relevant

Reply:
The difference lies in the cardano-api which defines both `PlutusScript` and `Script`, which are used kind of interchangeably as far as I could tell. I haven't particularly search for a rationale but maybe that a discrepancy we can resolve upstream?

Reply:
Possibly, we could also makes it a typeclass `ToPlutusScript` and choose a target based on the context..